examples/ensemble/plot_forest_iris.py

```diff
@@ -90,7 +89,6 @@
         X = (X - mean) / std
 
         # Train
-        clf = clone(model)
```

Comment:
I think the code was supposed to be:

```py
clf = clone(model)
clf.fit(...)
```

This is to ensure that `model` stays unfitted and unchanged throughout the loop.

Reply:
@lesteve
Thanks for the review but the problem seems not simple. In the loop, the original author use 'model' instead of 'clf' to do most things(e.g., prediction). 'clf' is only used to calculate the score. Personally, I think it will be much better to modify the example according to your suggestion. Do you agree?